docs/source/examplecleanup.rst

```diff
@@ -84,11 +135,11 @@ An End-to-End example walkthrough of a tartufo scan and the process of purging t
        ! [remote rejected] refs/pull/7/head -> refs/pull/7/head (deny updating a hidden ref)
        ! [remote rejected] refs/pull/8/head -> refs/pull/8/head (deny updating a hidden ref)
        ! [remote rejected] refs/pull/9/head -> refs/pull/9/head (deny updating a hidden ref)
-      error: failed to push some refs to 'git@${GITHUBADDRESS}:yourproject/yourrepo.git'
+      error: failed to push some refs to 'git@GITHUBADDRESS:yourproject/yourrepo.git'
       (.venv) you@LTDV-you:~/tartufo/yourrepo.git$
 
 
-   If you get the above error; It might actually be ok, re-run tartufo. Only if there are results that are not clean continue:
+   If you get the above error; It might actually be ok, re-run tartufo from master. Only if there are results that are not clean continue. Please note, this solution will remove PR history (but not commit history):
```

Comment:
Minor pedantic grammar update 😄 
```suggestion
   If you get the above error, it might actually be ok; just re-run tartufo from master. Only continue if there are results that are not clean. Please note, this solution will remove PR history (but not commit history):
```

Reply:
👍  modifying